hexya/server/modules.go

```diff
@@ -83,6 +83,11 @@ func LoadDemoRecords() {
 // LoadTranslations loads all translation data from the PO files in the 'i18n' directory
 // into the translations registry.
 func LoadTranslations(langs []string) {
+	for i, lang := range langs {
+		if lang == "ALL" {
```

Comment:
Déjà vu dans Boostrap => refactorise le code (+ sortir la condition de la boucle for)